app/src/userpages/components/StreamPage/Show/ConfigureView/index.jsx

```diff
@@ -57,17 +57,16 @@ export class ConfigureView extends Component<Props, State> {
     }
 
     componentDidUpdate(prevProps: Props) {
-        if (this.validFieldProps(prevProps) && this.validFieldProps(this.props) &&
-        // $FlowFixMe
-        (this.props.stream.config.fields !== prevProps.stream.config.fields)) {
+        const { config = {} } = this.props.stream || {}
+        const { config: prevConfig = {} } = prevProps.stream || {}
+
+        if (config.fields && prevConfig.fields && config.fields !== prevConfig.fields) {
```

Comment:
I think we can/should actually start using this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
and while we're at it instead of `|| {}` use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_Coalescing_Operator

Reply:
I'll leave it cause I don't wanna spend more time here, but yeah, let's start using them!